spec/controllers/orders_controller_spec.rb

```diff
@@ -20,4 +24,40 @@
       end
     end
   end
+
+  describe 'GET index' do
+    let(:params) { {} }
+
+    before { get :index, params: params }
+
+    context 'without params' do
+      it { expect(response).to render_template :index }
+    end
+
+    context 'with status parameter 2' do
+      let(:params) { { status: 2 } }
+
+      it { expect(response).to render_template :index }
+    end
+
+    context 'with status parameter 3' do
+      let(:params) { { status: 3 } }
+
+      it { expect(response).to render_template :index }
+    end
+
+    context 'with status parameter 4' do
+      let(:params) { { status: 4 } }
+
+      it { expect(response).to render_template :index }
+    end
```

Comment:
What do you test here?
Maybe you need to add a check that some value is store to an instance variable or failure flow?